docs/devops.md

```diff
@@ -4,110 +4,185 @@ title: DevOps
+  </iframe>
 </figure>
 
 * * *
+### Kubernetes
+
+Kubernetes es un sistema de código libre para la automatización del despliegue, ajuste de escala y manejo de aplicaciones en contenedores​ que fue originalmente diseñado por Google y donado a la Cloud Native Computing Foundation. Soporta diferentes entornos para la ejecución de contenedores, incluido Docker.
+
+A continuación, puede encontrar un conjunto de tutoriales muy convenientes para aprender kubernetes desde cero:
 
-### **Kubernetes** en 5 minutos
+#### **Kubernetes en 5 minutos**
 
 <figure class="video_container">
-  <iframe width="100%" height="315" src="https://www.youtube.com/embed/PH-2FfFD2PU" frameborder="0" allowfullscreen="true"> </iframe>
+  <iframe width="100%" height="315" src="https://www.youtube.com/embed/PH-2FfFD2PU" frameborder="0" allowfullscreen="true">
+  </iframe>
 </figure>
 
 * * *
 
-### Kubernetes vs. Docker: No es una pregunta de ambos o uno
+#### **Kubernetes vs.Docker: no es una cuestión de uno o el otro**
 
 <figure class="video_container">
-  <iframe width="100%" height="315" src="https://www.youtube.com/embed/2vMEQ5zs1ko" frameborder="0" allowfullscreen="true"> </iframe>
+  <iframe wwidth="100%" height="315" src="https://www.youtube.com/embed/2vMEQ5zs1ko" frameborder="0" allowfullscreen="true">
+  </iframe>
 </figure>
 
-## Kubernetes en EOS Costa Rica
+* * *
 
-A continuación puedes encontrar un conjunto de sencillos tutoriales para aprender kubernetes desde cero:
+## Kubernetes en EOS Costa Rica
 
-https://kubernetes.io/es/docs/tutorials/kubernetes-basics/
+[Aprenda los conceptos básicos de Kubernetes](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
 
-Par probar los anteriores tutoriales en un ambiente real, podrías utilizar nuestra instalación de kubernetes sobre Openstack. 
-Si así lo prefieres, también puedes crear un cluster local en tu propia máquina: 
+Para poder probar estos tutoriales en un entorno real, puede probar nuestra instalación de Kubernetes
+en Openstack. Si lo prefiere, también puede crear una configuración local en su propia máquina:
 
-https://computingforgeeks.com/how-to-install-minikube-on-ubuntu-debian-linux/. 
+[Instalar Minikube](https://computingforgeeks.com/how-to-install-minikube-on-ubuntu-debian-linux/)
 
-Pídele a alguno de nuestros administradores accesso al dashboard de nuestra instalación de Openstack y el projecto Sandbox. 
-Luego puedes seguir los siguientes pasos:
+Solicite a los administradores acceso a nuestro panel de instalación Openstack y al Sandbox
+proyect. Entonces puedes seguir los siguientes pasos:
 
-- Antes que nada conectate al nodo maestro usando tu herramienta de SSH:
+- Primero que nada conéctese al nodo maestro usando la herramienta SSH:
 
-```
+```shell
 ssh -i key.pem user@master-sandbox
 ```
 
-- Obtén información acerca del cluster:
+- Obtener información sobre el clúster:
 
-```
+```shell
 kubectl cluster-info
 ```
+- Muestrar nodos disponibles actualmente en el clúster:
```

Comment:
> Muestrar

Is that right?

Reply:
![image](https://user-images.githubusercontent.com/76976401/106071788-f6824d80-60d4-11eb-9511-e0f6a6dd2a72.png)